examples/examples/multisig.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! To run this example, a local polkadot node should be running. Example verified against substrate latest-38a955ba4a0.
+//!
+//! E.g.
+//! ```bash
+//! curl "https://releases.parity.io/substrate/x86_64-debian:stretch/latest/substrate/substrate" --output /usr/local/bin/substrate --location
+//! substrate --dev --tmp
+//! ```
```

Comment:
Is this the only example using Substrate? I think just for consistency with other examples maybe it's best to stick to Polkadot?

Reply:
The Polkadot node does not contain the `Assets` pallet, at least I didn't see it in the metadata of `wss://rpc.polkadot.io:443`

Reply:
Oh, that's ok; we don't need to specifically use this Assets call as the example; in https://github.com/paritytech/subxt/pull/714/files I used a balance transfer so perhaps we could standardise and both just do that?

Reply:
Also I think I'll rename my example to `dynamic_multisig.rs` to align with yours :)